modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -6,9 +6,18 @@
 [id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
 = Configuring the PROXY protocol for an Ingress Controller
 
-A cluster administrator can configure https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[the PROXY protocol] when an Ingress Controller uses either the `HostNetwork` or `NodePortService` endpoint publishing strategy types. The PROXY protocol enables the load balancer to preserve the original client addresses for connections that the Ingress Controller receives. The original client addresses are useful for logging, filtering, and injecting HTTP headers. In the default configuration, the connections that the Ingress Controller receives only contain the source address that is associated with the load balancer.
```

Comment:
Since [OCPBUGS-3559](https://issues.redhat.com/browse/OCPBUGS-3559) was fixed in OCP 4.12.0 and backported to OCP 4.11.40 as [OCPBUGS-33700](https://issues.redhat.com/browse/OCPBUGS-33700), PROXY protocol can also be enabled for the `Private` endpoint publishing strategy type.  Would you like me to file a separate bug report for that?

Reply:
I filed [OCPBUGS-36292](https://issues.redhat.com/browse/OCPBUGS-36292) for this issue.